nbs/flags_page.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>Generalize this language  "if one specific" -&gt; "If specific cells"</p><p> </p><p>Also remove the word "just"</p><p> </p><p>Provide a link to some discussion on what <strong> </strong><code><strong>all</strong></code><strong> </strong> is  for as most people don't know what that  is.  This is a good link http://xion.io/post/code/python-all-wild-imports.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/418/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>